exporter.go

```diff
@@ -49,28 +53,33 @@ func (e *Exporter) ExportSpans(ctx context.Context, spans []sdktrace.ReadOnlySpa
 		return nil
 	}
 
-	var lumigoSpans []telemetry.Span
 	e.encoderMu.Lock()
 	defer e.encoderMu.Unlock()
+
 	for _, span := range spans {
 		mapper := transform.NewMapper(e.context, span, logger)
 		lumigoSpan := mapper.Transform()
+		e.spansTotalSizeBytes += int(reflect.TypeOf(lumigoSpan).Size())
+		// maximum size to lumigoSpans RD-7826
+		maxSizeOfSpansString := getenv("MAX_SIZE_FOR_REQUEST", 1024*500)
```

Comment:
Can you extract it to a method? I think it will be used in more places in the future?